src/go/rpk/pkg/cli/cmd/start.go

```diff
@@ -140,7 +143,10 @@ func NewStartCommand(
 				rpcAddr,
 				os.Getenv("REDPANDA_RPC_ADDRESS"),
 			)
-			rpcServer, err := parseAddress(rpcAddr)
+			rpcServer, err := parseAddress(
+				rpcAddr,
+				conf.Redpanda.RPCServer.Port,
```

Comment:
did you mean to do config.Default()....?

Reply:
I did! Good catch, thanks.